src/Resources/views/Collector/cassandra.html.twig

```diff
@@ -16,9 +16,9 @@
 
 {% block menu %}
     <span class="label">
-    <span class="icon">{% spaceless %}
+    <span class="icon">{% apply spaceless %}
```

Comment:
```suggestion
    <span class="icon"><img ... /></span>
```

using `apply spaceless` is a BC break for those using an old Twig version, so we'd prefer to simply remove the need for `spaceless` by simply removing the spaces "manually"

Reply:
@Oliboy50 I have commited your suggestion. Already is all right ?